pages/contribute.md

```diff
@@ -6,44 +6,58 @@ main_nav_id: contribute
+and we will try to fix them in the next EAP build. This way, you can get bug fixes earlier than the standard Kotlin release cycle.
 
-If you want to contribute to the code, go to GitHub and check out the latest version and follow the instructions on how to build Kotlin from source. After that, you can start picking
-some [pending tasks](https://youtrack.jetbrains.com/issues/KT?q=%23Unresolved+) on the issue tracker. Make sure you look for issues tags with [Up For Grabs](https://youtrack.jetbrains.com/issues/KT?q=%23Unresolved+tag%3A+%7BUp+For+Grabs%7D) as
-these are some of the easier ones to get started with.
+## Contribute to the compiler, standard library, and tooling
 
-### Contributing to the documentation or web site
+If you want to contribute to the Kotlin compiler, standard library, or tooling, go to [GitHub](https://github.com/jetbrains/kotlin), check out the latest Kotlin version,
+ and follow the [instructions on how to contribute to the code](https://github.com/JetBrains/kotlin/blob/master/docs/contributing.md).
+ 
+You can help us by completing [open tasks](https://youtrack.jetbrains.com/issues/KT?q=tag:%20%7BUp%20For%20Grabs%7D%20and%20State:%20Open). 
+Please stay in touch with us because we may have questions and comments on your changes. Otherwise, we won't be able to incorporate them. 
 
-We need a lot more documentation. If you're interested in collaborating, please feel free to check out the source code [to this site on GitHub](https://github.com/jetbrains/kotlin-web-site) and send a Pull Request. The site
-is built using Markdown and Jekyll. 
+## Contribute to other libraries and create your own
 
-### Contributing with tutorials or videos
+Kotlin ships with the standard library providing core capabilities and a number of additional (kotlinx) libraries that extend its functionality. 
+Each kotlinx library is developed in a separate repo, has its own versioning and release cycle. 
 
-Have you created a tutorial or video on Kotlin? Please let us know. We'd be more than happy to have it featured under [Community Content](http://kotlinlang.org/docs/resources.html).
+If you want to contribute to the standard library, follow [the standard process for contributing](https://github.com/JetBrains/kotlin/blob/master/docs/contributing.md).
 
-### Presentations
+If you want to contribute to a kotlinx library, follow the contribution process described for each library, such as [kotlinx.coroutines](https://github.com/Kotlin/kotlinx.coroutines),
+ [kotlinx.serialization](https://github.com/Kotlin/kotlinx.serialization/blob/master/CONTRIBUTING.md), and [ktor](https://github.com/ktorio/ktor/blob/master/CONTRIBUTING.md).
 
-If you've given or are giving presentations on Kotlin, let us know. We'll [feature it on the list](http://kotlinlang.org/docs/events.html)
+If you have a library that could be useful for others, let us know via <feedback@kotlinlang.org>.
```

Comment:
If you have a library that could be useful for other Kotlin developers

Reply:
done